packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -28,7 +28,19 @@ const String kFlutterEnginePackageName = 'sky_engine';
 class FlutterCommandRunner extends CommandRunner {
   FlutterCommandRunner({ bool verboseHelp: false }) : super(
     'flutter',
-    'Manage your Flutter app development.'
+    'Manage your Flutter app development.\n'
+      '\n'
+      'Common actions:\n'
+      '\n'
+      '  flutter create <output directory>\n'
+      '    Create a new Flutter project in the specified directory.\n'
+      '\n'
+      '  flutter analyze [options] [<target>]\n'
+      '    Analyze Flutter project\'s Dart code.\n'
```

Comment:
I'd skip analyze and just have "create" and "run".  Most people are going to use IDEs with built-in analyzer support, which means they won't need to run "analyze" from the command line too often.